priv/repo/migrations/20240609092546_add_errors_to_trigger_kafka_messages.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Lightning.Repo.Migrations.AddProcessingDataToTriggerKafkaMessages do
+  use Ecto.Migration
+
+  def change do
+    alter table(:trigger_kafka_messages) do
+      add :processing_data, :map, default: %{}
+    end
+  end
+end
```

Comment:
better to move the change to the `CreateTriggerKafkaMessages` to have a single file for `:trigger_kafka_messages`.

Reply:
@jyeshe Unfortunately, people other than you and I will already have run this migration, so I would prefer to just keep it in place so that they don't end up with an orphaned migration and no idea where it came from.

Reply:
As we are closing a version for staging this shouldn't be a concern but it's okay.